packages/clay-shared/src/useFocusManagement.ts

```diff
@@ -0,0 +1,224 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {useEffect, useRef} from 'react';
+
+type FocusManager = {
+	key: string;
+	portal?: boolean;
+	value: HTMLElement;
+};
+
+const TAB_KEY_CODE = 9;
+const FOCUSABLE_ELEMENTS = [
```

Comment:
Did you copy this list of elements from somewhere? If so, I think it'd be helpful to document that here in a comment.

Reply:
In fact it was a mix, some that I added and others that I picked up from somewhere that I do not remember anymore 😅, this ended up coming from the v2 version I was using for Modal.